core/src/main/resources/MySQL.yml

```diff
@@ -0,0 +1,7 @@
+MySQL:
+  IP: localhost
```

Comment:
Why is the MySQL.yml brought back? I thought those settings were moved into config.yml

Reply:
As I already said before.. the library I used require an InputStream to inizialize the file, but of you look at the First lines for VelocityMethods mysql file Will be loader ONLY if the file exists